### PR TITLE
A more proper usage of "is" helpers and code clean up.

### PR DESCRIPTION
**Pull request by Ghommie**
:cl:
code: Better usage of "is" helpers across the code.
/:cl:

Kinda of what it tells on the tin.
Nothing extraordinary, is_helpers.dm, containing many (not all) of them.

### CODE REVIEW

**Comment by DominikPanic** [comment]
Conflicts.

**Comment by Rohesie** [comment]
Conflicts.

File: code/game/objects/structures/kitchen_spike.dm [hunk 1]
```diff
@@ -18,7 +18,7 @@
 	attackby(obj/item/grab/G, mob/user)
 		if(!istype(G, /obj/item/grab))
 			return
-		if(istype(G.grabbed_thing, /mob/living/carbon/monkey))
+		if(istype(G.grabbed_thing))
```
Comment (DominikPanic) [inline code/game/objects/structures/kitchen_spike.dm]: This will runtime on anything but mob/living.

File: code/game/turfs/snow.dm [hunk 2]
```diff
@@ -73,7 +73,7 @@
 			var/mob/living/carbon/C = AM
 			var/slow_amount = 0.75
 			var/can_stuck = 1
-			if(istype(C, /mob/living/carbon/Xenomorph)||isYautja(C))
+			if(istype(C, /mob/living/carbon/Xenomorph)||isyautja(C))
```
Comment (DominikPanic) [inline code/game/turfs/snow.dm]: C'mon :c

**Pull request merged**